### PR TITLE
fix: check bech32 address format in bank query

### PR DESCRIPTION
**Pull request by egonspace**
## Description

closes: #362 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by dudong2 — COMMENTED** [review]
Is it okay if everything in changelog.md deleted?

File: RELEASE_CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [v0.43.1](https://github.com/line/lbm-sdk/releases/tag/v0.43.1)
+
+### Bug Fixes
+* (distribution) [\#364](https://github.com/line/lbm-sdk/pull/364) Emit error in tx withdraw-all-rewards when the signer has no delegations
```
Comment (0Tech) [inline RELEASE_CHANGELOG.md:6]:
```suggestion
* (distribution) [\#364](https://github.com/line/lbm-sdk/pull/364) Force genOrBroadcastFn even when max-msgs != 0
```
I've updated the title of my PR. Sorry for the interrupt.

**Review by 0Tech — APPROVED** [review]

**Comment by egonspace** [comment]
> Is it okay if everything in changelog.md deleted?

Yes. I intended that `CHANGELOG` of a release branch contains empty.

File: RELEASE_CHANGELOG.md (hunk 1 above)
Comment (egonspace) [reply RELEASE_CHANGELOG.md]: Okay, I'll apply it.

**Review by iproudhon — APPROVED** [review]
LGTM

**Pull request merged**